src/paramsUi.cpp

```diff
@@ -390,6 +390,8 @@ class ParamsDialog {
 		char rawText[30];
 		if (GetDlgItemText(dialog, ID_PARAM_VAL_EDIT, rawText, sizeof(rawText)) == 0)
 			return;
+		if (this->param->getValueForEditing() == string(rawText))
```

Comment:
Avoid constructing string.
```suggestion
		if (this->param->getValueForEditing().compare(rawText) == 0)
```